validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 11 | Confirm resolution notifications sent | | Resolution notifications received          |
+
+### Backup Target Configuration
+
+Test backup target setup and configuration for S3 and NFS storage through Rancher interface.
+
+| Step Number | Action | Data | Expected Result                    |
+|------|--------|--------|-------------------------------------|
+| 1 | Access Longhorn Settings through Rancher → Longhorn | | Longhorn settings page displays     |
+| 2 | Configure S3 backup target: `s3://test-bucket@us-west-2/longhorn/` | | S3 backup target configured         |
+| 3 | Create Kubernetes secret with S3 credentials | | Secret created with AWS access keys |
+| 4 | Set "Backup Target Credential Secret" to created secret | | Credential secret configured        |
+| 5 | Click "Test Connection" for backup target | | Connection test passes successfully |
+| 6 | Create test volume with sample data | | Volume created and data written     |
+| 7 | Navigate to Volume in Longhorn UI and click "Create Backup" | | Backup creation dialog opens        |
+| 8 | Create backup with name "test-backup-001" |  | Backup process initiates            |
+| 9 | Monitor backup progress until completion | | Backup completes successfully       |
+| 10 | Verify backup appears in backup list | | Backup listed with correct metadata |
+| 11 | Test backup integrity verification | | Backup integrity check passes       |
+
+### Cross-Cluster Disaster Recovery
+
+Test disaster recovery volume functionality between separate Rancher-managed clusters with shared backup storage.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Configure same S3 backup target on both source and target clusters | | Both clusters use shared backup storage |
+| 2 | Create volume with test data in source cluster | | Volume created with identifiable test data |
+| 3 | Set up recurring backup schedule (every 30 minutes) | | Recurring backup configured |
+| 4 | Wait for initial backup completion | | Backup appears in source cluster |
+| 5 | Access target cluster Longhorn UI → Backup → Volume | | Backup from source cluster visible |
+| 6 | Select backup and click "Create Disaster Recovery Volume" | | DR volume creation dialog opens |
+| 7 | Configure DR volume with name "dr-test-volume" | | DR volume configuration set |
+| 8 | Create DR volume and verify "Standby" state | | DR volume shows as standby |
+| 9 | Monitor incremental synchronization progress | | DR volume stays synchronized |
+| 10 | Simulate source cluster failure (power off nodes) | | Source cluster becomes unavailable |
+| 11 | Activate DR volume in target cluster | | DR volume activation succeeds |
+| 12 | Deploy application using activated DR volume | | Application successfully uses DR volume |
+| 13 | Verify data integrity matches source cluster | | All test data present and correct |
+
+### Node Drain Operation via Rancher
```

Comment:
imo would remove `operation via rancher` as its implied

Reply:
This title differentiates between the Rancher UI and Longhorn UI.

Reply:
aren't all of the tests here only for rancher UI? or are some for longhorn UI?